### DISCUSSION TITLE
Consil pro bono?

### DISCUSSION THREAD

**Discussion by emmanuelbernard** [commonhaus/foundation#15] [Q&A]
It's not clearly said but are council member pro bono? Expected presence and involvment vs sitting and doing nothing?

**Comment by ebullient** [comment]
I would like to frame this as a more approachable/cooperative body, but the Commonhaus Council is essentially the Board of Directors. There is absolutely a requirement of active participation, ensuring council activities are as efficient as possible will be a goal to prevent this from being burdensome.

Board members are usually volunteer / pro-bono in software foundations. I want to avoid some aspects of usual formal top-down activities, but allocation of resources for the benefit of members and member projects would be a key topic. I would like something more streamlined than what Apache does, though the premise is the same: https://www.apache.org/foundation/governance/board.html: 

> Article V. of the ASF bylaws specifies a Board of nine Directors, elected annually by the ASF Membership. The Apache Board is much like the board of directors of any other corporation, responsible for -- management of the corporate assets (funds, intellectual property, trademarks, and support equipment) and allocation of corporate resources to projects.

The [Gnome Foundation](https://wiki.gnome.org/FoundationBoard) also has a useful definition of the requirements for their board.

If/when foundation gets big enough to want paid staff, the evaluation follows criteria like this: https://wiki.gnome.org/CompensationCommittee (this applies to Linux Foundation, OpenJS Foundation, etc). The question is if/when the Foundation gets big enough to want to pay staff salaries (and HR, etc). It requires a sustained revenue stream and the determination that Foundation funds are best used for that purpose.

Note: Accounting will be delegated to pick-a-fiscal-host. Most fiscal hosts do not require up-front/recurring payments, they instead take a percentage on inbound funds (which I find reasonable, as it doesn't introduce a recurring fee that requires a steady income stream.. that's super important for bootstrapping something new) 






Note: Accounting will be delegated to pick-a-fiscal-host. Most of those available do not require up-front/recurring payments, they instead take a percentage on inbound funds (which I find reasonable, as it doesn't introduce a recurring fee that requires a steady income stream.. that's super important for bootstrapping something new)

**Comment by kenfinnigan** [comment]
This question is now resolved in the by-laws. If anything above is not resolved, please open a PR with suggested changes to the by-laws/docs.
